src/rws_interface.cpp

```diff
@@ -82,6 +82,29 @@ RWSInterface::StaticInfo RWSInterface::collectStaticInfo()
   return static_info;
 }
 
+std::vector<cfg::sys::PresentOption> RWSInterface::getCFGPresentOptions()
+{
+  std::vector<cfg::sys::PresentOption> result;
+
+  RWSClient::RWSResult rws_result = rws_client_.getConfigurationInstances(Identifiers::SYS,
+                                                                          Identifiers::PRESENT_OPTIONS);
+
+  std::vector<Poco::XML::Node*> node_list = xmlFindNodes(rws_result.p_xml_document, XMLAttributes::CLASS_CFG_IA_T_LI);
+
+  for (size_t i = 0; i < node_list.size(); i+=2)
```

Comment:
Because it was based on a previous implementation that wasn't so fleshed out.

I have reworked it to be more like the other similar PRs in 72d5142, though this now need #113 to be merged first.